src/index.ts

```diff
@@ -57,6 +57,7 @@ export function getSaveGamesFolder(): string {
  * Returns path for type or undefined if name is invalid
  * @param name folder type (see README)
  */
+type Folder = 'home' | 'appData' | 'appdata' | 'userData' | 'desktop' | 'documents' | 'downloads' | 'music' | 'pictures' | 'videos' | 'cache' | 'savegames';
 export default function getPath(name: string): string|undefined {
```

Comment:
I think you have to replace `name: string` with `name: Folder` to use the type declared above